lib/models/os.ts

```diff
@@ -287,17 +285,13 @@ const getOsModel = function (
 				getOsVersionReleaseLine(release.phase, strippedVersion, appTags) ??
 				undefined;
 
-			// TODO: Don't append the variant and sent it as a separate parameter when requesting a download when we don't use /device-types anymore and the API and image maker can handle it. Also rename `rawVersion` -> `versionWithVariant` if it is needed (it might not be needed anymore).
-			// The version coming from release tags doesn't contain the variant, so we append it here
 			return {
 				...release,
 				// TODO: Drop the explicit assignment once we no longer convert empty strings to undefined
 				variant,
 				osType: appTags.osType,
 				line,
 				strippedVersion,
-				// TODO: Drop in the next major
-				rawVersion: release.raw_version,
 				basedOnVersion: tagMap[BASED_ON_VERSION_TAG_NAME] ?? strippedVersion,
 				// TODO: Drop in the next major
```

Comment:
I decided to postpone this (along with some others) since out modules are still using it and the PR became large enough already. Maybe we can drop it on March.